src/Symfony/Component/Mailer/composer.json

```diff
@@ -35,7 +35,8 @@
         "symfony/sendgrid-mailer": "^4.4|^5.0"
     },
     "conflict": {
-        "symfony/http-kernel": "<4.4"
+        "symfony/http-kernel": "<4.4",
+        "symfony/mime": "<5.2"
```

Comment:
this can be removed since mime is in "require" and not in "required-dev"

Reply:
Good catch